tutorials/dataframe/df106_HiggsToFourLeptons_withVariations.py

```diff
@@ -174,22 +248,16 @@
 legend.SetTextSize(0.04)
 legend.SetTextAlign(32)
 legend.AddEntry(h_data, "Data" ,"lep")
-legend.AddEntry(h_higgs, "Higgs", "f")
-legend.AddEntry(h_zz, "ZZ", "f")
-legend.AddEntry(h_other, "Other", "f")
+legend.AddEntry(h_higgs, "Higgs MC", "f")
+legend.AddEntry(h_zz, "ZZ MC", "f")
+legend.AddEntry(h_other, "Other MC", "f")
+legend.AddEntry(((histos_mc)["nominal"]), "Total MC", "l")
+legend.AddEntry(((histos_mc)["weight:up"]), "Total MC: variations up", "l")
+legend.AddEntry(((histos_mc)["weight:down"]), "Total MC: variations down", "l")
 legend.Draw("SAME")
 
-# Add ATLAS label
-text = ROOT.TLatex()
-text.SetNDC()
-text.SetTextFont(72)
-text.SetTextSize(0.045)
-text.DrawLatex(0.21, 0.86, "ATLAS")
-text.SetTextFont(42)
-text.DrawLatex(0.21 + 0.16, 0.86, "Open Data")
-text.SetTextSize(0.04)
-text.DrawLatex(0.21, 0.80, "#sqrt{s} = 13 TeV, 10 fb^{-1}")
+ATLAS_label()
```

Comment:
```suggestion
draw_atlas_label()
```
Another suggestion for an alternative name. Usually, functions names are expressions of *doing* something. Like this it is easier to distinguish them from variables.